index.js

```diff
@@ -98,67 +97,261 @@ enum PageType {
-    # creates a Questionnaire along with an initial Section and Page
-    createQuestionnaire(title: String!, description: String, theme: String!, legalBasis: LegalBasis!, navigation: Boolean, surveyId: String!) : Questionnaire
-    updateQuestionnaire(id: Int!, title: String, description: String, theme: String, legalBasis: LegalBasis, navigation: Boolean, surveyId: String) : Questionnaire
-    deleteQuestionnaire(id: Int!) : Questionnaire
-
-    createSection(title: String!, description: String, questionnaireId: Int!) : Section
-    updateSection(id: Int!, title: String, description: String) : Section
-    deleteSection(id: Int!) : Section
-    createGroup(title: String!, description: String, questionnaireId: Int!) : Section @deprecated(reason: "use 'createSection' instead")
-    updateGroup(id: Int!, title: String, description: String) : Section @deprecated(reason: "use 'updateSection' instead")
-    deleteGroup(id: Int!) : Section @deprecated(reason: "use 'deleteSection' instead")
-
-    createPage(title: String!, description: String, sectionId: Int!) : Page
-    updatePage(id: Int!, title: String!, description: String) : Page
-    deletePage(id: Int!) : Page
-
-    createQuestionPage(title: String!, description: String, guidance: String, sectionId: Int!) : QuestionPage
-    updateQuestionPage(id: Int!, title: String, description: String, guidance: String) : QuestionPage
-    deleteQuestionPage(id: Int!) : QuestionPage
-
-    createAnswer(description: String, guidance: String, label: String, qCode: String, type: AnswerType!, mandatory: Boolean!, questionPageId: Int!) : Answer
-    updateAnswer(id: Int!, description: String, guidance: String, label: String, qCode: String, type: AnswerType, mandatory: Boolean) : Answer
-    deleteAnswer(id: Int!) : Answer
-    
-    createOption(label: String, description: String, value: String, qCode: String, childAnswerId: Int, answerId: Int!) : Option
-    updateOption(id: Int!, label: String, description: String, value: String, qCode: String, childAnswerId: Int) : Option
-    deleteOption(id: Int!) : Option
+  # creates a Questionnaire along with an initial Section and Page
+  createQuestionnaire(
+    title: String
+    description: String
+    theme: String
+    legalBasis: LegalBasis
+    navigation: Boolean
+    surveyId: String
+    input: CreateQuestionnaireInput
+  ): Questionnaire
+  updateQuestionnaire(
+    id: Int
+    title: String
+    description: String
+    theme: String
+    legalBasis: LegalBasis
+    navigation: Boolean
+    surveyId: String
+    input: UpdateQuestionnaireInput
+  ): Questionnaire
+  deleteQuestionnaire(id: Int, input: DeleteQuestionnaireInput): Questionnaire
+  createSection(
+    title: String
+    description: String
+    questionnaireId: Int
+    input: CreateSectionInput
+  ): Section
+  updateSection(
+    id: Int
+    title: String
+    description: String
+    input: UpdateSectionInput
+  ): Section
+  deleteSection(id: Int, input: DeleteSectionInput): Section
```

Comment:
Sorry to be a pain, but can we just get some white space between each of these. Just makes it a bit easier to read.